packages/quantic/force-app/main/default/lwc/quanticPager/e2e/fixture.ts

```diff
@@ -0,0 +1,63 @@
+import {PagerObject} from './pagerObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+
+const pagerUrl = 's/quantic-pager';
+
+interface PagerOptions {
+  numberOfPages: number;
+}
+type QuanticPagerE2EFixtures = {
+  pager: PagerObject;
+  search: SearchObject;
+  options: Partial<PagerOptions>;
+};
+
+type QuanticPagerE2ESearchFixtures = QuanticPagerE2EFixtures & {
+  urlHash: string;
+};
+
+type QuanticPagerE2EInsightFixtures = QuanticPagerE2ESearchFixtures & {
+  insightSetup: InsightSetupObject;
+};
+
+export const testSearch = quanticBase.extend<QuanticPagerE2ESearchFixtures>({
+  options: {},
+  urlHash: '',
+  search: async ({page}, use) => {
+    await use(new SearchObject(page, searchRequestRegex));
+  },
+  pager: async ({page, options, configuration, search, urlHash}, use) => {
+    await page.goto(urlHash ? `${pagerUrl}#${urlHash}` : pagerUrl);
+    configuration.configure(options);
```

Comment:
Is there value in being explicit here you think?


```suggestion
    configuration.configure({...options, useCase: useCaseEnum.search});
```

Reply:
Just for clarity sake but I have no strong opinions,